capstone/src/components/User/Adventurer/FindAdventure/FindAdventure.jsx

```diff
@@ -0,0 +1,34 @@
+import './FindAdventure.css';
+import { useState } from 'react';
+import Map from './Map/Map';
+
+export default function FindAdventure() {
+  const [displayExperience, setDisplayExperience] = useState('map');
+  return (
+    <div className="findAdventure">
+      <h2>Find your next adventure</h2>
+      <ExperienceDisplayer
+        displayExperience={displayExperience}
+        setDisplayExperience={setDisplayExperience}
+      />
+      <Map />
+    </div>
+  );
+}
+
+export function ExperienceDisplayer({ displayExperience, setDisplayExperience }) {
+  if (displayExperience === 'map') {
+    return (
+      <div className="experienceDisplayer">
+        <button type="button" value="map" className="chosenDisplay" disabled>Map</button>
+        <button type="button" value="list" className="notChosenDisplay" onClick={(e) => setDisplayExperience(e.target.value)}>List</button>
+      </div>
+    );
+  }
+  return (
+    <div className="experienceDisplayer">
```

Comment:
Oh! I implemented that one on the userMap branch, since it was more related to that than with the logIn, but thanks a lot for letting me know! I will try to be more organize with what I commit